src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -226,13 +221,11 @@ public void RemoveAllButThisTab(string tabName)
          if (Sheets.Keys.Any(k => k != tabName))
          {
             DataSheet currentAlreadyLoaded = null;
-            if (Sheets.Keys.Contains(tabName))
-               currentAlreadyLoaded = Sheets[tabName];
+            if (Sheets.Keys.Contains(tabName)) currentAlreadyLoaded = Sheets[tabName];
 
             Sheets.Clear();
-            if (currentAlreadyLoaded != null)
-               Sheets.Add(tabName, currentAlreadyLoaded);
-            
+            if (currentAlreadyLoaded != null) Sheets.Add(tabName, currentAlreadyLoaded);
```

Comment:
no this needs to be all undone here. We do not want statements on one line like this at all.
Please undo this commit